google/ggrc-core
# GGRC-5270,5328 Process rows on import one by one

# Issue description

Converters for all blocks and rows are instantiated at once in the beginning of import, eating up all memory before meaningful work is done.

# Steps to test the changes

Import a big enough file (1k rows) and check that the memory consumption is close to constant and doesn't exceed 700 megabytes during processing (some leak is inevitable due to how we log slow SQLs etc.).

# Solution description

The method that creates row converters is rewritten as a generator and the logic broken by this change is fixed. :)

# Sanity checklist

- [ ] My changes are covered by tests. - existing tests have good coverage already



# PR Review checklist
